readme.md

```diff
@@ -32,14 +34,27 @@ console.log(chalkTemplate`
 	In {bold ${miles} miles}, there are {green.bold ${calculateFeet(miles)} feet}.
 `);
 
-
 console.log(chalkTemplate`
 	There are also {#FF0000 shorthand hex styles} for
 	both the {#ABCDEF foreground}, {#:123456 background},
 	or {#ABCDEF:123456 both}.
 `);
 ```
 
+For printing to standard error (stderr):
+
+```js
+import {chalkStderr} from 'chalk-template';
```

Comment:
For consistency, this should be `chalkTemplateStderr`.

Reply:
Ehh I suppose. That's only convention though, because the default can be called whatever. When I use this library I don't use regular `chalk`, so I usually `import chalk from 'chalk-template'`. I suppose it's just a preference, but I'll change it anyway.